consensus/src/quorum_store/batch_generator.rs

```diff
@@ -41,14 +41,13 @@ pub struct BackPressure {
 }
 
 pub struct BatchGenerator {
-    epoch: u64,
```

Comment:
@sasha8 why did we have epoch here?

Reply:
Looks like it's now getting from `latest_logical_time`. I'll probably revive it for the now-LogicalTime-based expirations 😆